app/Http/Controllers/SystemController.php

```diff
@@ -57,4 +59,29 @@ public function facilityList(): JsonResource
     {
         return FacilityResource::collection(Facility::query()->get());
     }
+
+    #[OA\Get(
+        path: '/api/v1/system/dictionary/list',
+        description: new PermissionDescribe(Permission::any),
+        summary: 'All facilities',
+        tags: ['System'],
+        parameters: [new OA\Parameter(name: 'in', in: 'query')],
+        responses: [
+            new OA\Response(
+                response: 200, description: 'OK', content: new  OA\JsonContent(properties: [
+                new OA\Property(
+                    property: 'data', type: 'array', items: new OA\Items(
+                    ref: '#/components/schemas/DictionaryResource'
+                ),
+                ),
+            ])
+            ),
+        ]
+    )]
+    public function dictionaryList(): JsonResource
+    {
+        $dictionariesQuery = Dictionary::query();
+        $this->applyRequestIn($dictionariesQuery);
```

Comment:
rozwinąłbyś do czego to jest konkretnie potrzebne?

Reply:
szybkie filtrowanie po id: https://github.com/mblajek/Memo/blob/develop/help/api.md#get-list

nie wiem jak dużo tych słowników będzie, ale jeżeli będzie dużo, to można dać opcję pobrania wybranych

albo w swoim czasie dać endpoint do pobierania pozycji z takim filtrowaniem

na pewno będzie pobieranie osób, backend będzie tylko zwracał id członków spotkania